native/src/components/LocationButton.tsx

```diff
@@ -0,0 +1,58 @@
+import LocationIcon from '../assets/IconPlaceholder.svg'
```

Comment:
Perhaps rename `LocationButton` to `UserLocationButton` to avoid confusion with other usages of term location? Or something like `RequestLocationPermissionButton`?

Reply:
I decided to directly use the fitting material icons for this. If we do want custom buttons, we can still change this later.